docs/monster-ui/jsoneditor().md

```diff
@@ -0,0 +1,68 @@
+title: jsoneditor()
+
+# monster.ui.jsoneditor()
+The `monster.ui.jsoneditor()` method generates an instance of a JSON editor with a set of handy methods and properties.
+
+![](images/jsoneditor-preview.png)
+
+## Syntax
+```javascript
+monster.ui.jsoneditor(target[,options])
+```
+
+### Parameters
+
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`target` | A Jquery object which is automatically contain the JSON Editor | `jQuery` | |`true`
```

Comment:
The "Type" column already provides the type of this parameter, no need to repeat yourself.
```suggestion
`target` | Container for the JSON Editor widget. | `jQuery` | |`true`
```